### ISSUE TITLE
Add `lang` attribute to `<html>` tag

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1148]
## Problem

For a11y and HTML validation purposes, the `<html>` tag should have a `lang` attribute. See Google Lighthouse's error about this:

![image](https://user-images.githubusercontent.com/77128518/118337166-2ff29380-b4e1-11eb-8924-9853ea20c8a6.png)

## Solution

Based on the site-wide language that the user is using (English, Spanish, etc.), set the `lang` attribute on the `<html>` tag. [See details on the `lang` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang).

**Comment by neel-a-mehta** [comment]
This passes the Lighthouse test! Looks like this works.
